hsctl

```diff
@@ -132,8 +132,6 @@ preflight_hs() {
         # generate nginx configuration file
         if [ "${USE_SSL,,}" = true ]; then
             echo "*** Using SSL: USE_SSL = ${USE_SSL} ***"
-            # use https version of recaptcha
-            sed -i 's!'${HTTP_RECAPTCHA}'!'${HTTPS_RECAPTCHA}'!g' ${HS_PATH}/theme/templates/accounts/_signup_form.html
```

Comment:
@ironfroggy @mjstealey Removal of this sed statement that replace http with https for recaptcha URL in _signup_form.html in hsctl script causes issues we have been seeing on dev since this PR got merged. There are two options to fix this. One is to reinstate this statement in hsctl script, the other is to change the default http: recaptcha url in _signup_form.html in our codebase to be https rather than http. Which option do you think we should go with?